src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+
+```graphql
+{
+  currentUser {
+    account(id: YOUR_ACCOUNT_ID) {
+      aiIssues {
+        issues(cursor: "=ATN") {
+          issues {
+            issueId
+            priority
+            state
+            title
+          }
+          nextCursor
+        }
+      }
+    }
+  }
+}
+```
+
+ </Collapser>
+  <Collapser
+     className="freq-link"
+     id="filter"
+     title="Filter issues"
+   >
+
+Filter issues by entity guides, entity types, issues priority or issues states or any combination of those attributes.
+
+```graphql
+"Filter issues"
+input FilterIssues {
+  "entityGuids"
+  entityGuids: [EntityGuid!]
+  "entityTypes"
+  entityTypes: [String!]
+  "ids"
+  ids: [ID!]
+  "priority"
+  priority: [String!]
+  "states"
+  states: [IssueState!]
+}
+```
+
+Here `entityGuides` and `entityTypes` are entity attributes defined in [Learn about New Relic entities](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) document.
+
+* Issue priority could be one of the following: `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`
+* Issue states could be one of the following:  `CREATED`, `ACTIVATED`, `DEACTIVATED`, `CLOSED` 
+* `timeWindow` with start and end times in EpochMilliseconds format. If it is not provided the default value is the last 24 hours. 
+* And `cursor` as we have seen already 
+
+Let's find issues with priority `CRITICAL` and state `CLOSED`
+
+```graphql
+{
+  currentUser {
+    account(id: YOUR_ACCOUNT_ID) {
+      aiIssues {
+        issues(filter: {priority: "CRITICAL", states: CLOSED}) {
+          issues {
+            issueId
+            priority
+            state
+            title
+          }
+          nextCursor
+        }
+      }
+    }
+  }
+}
+```
+
+**Results**
+
+```json
+{
+  "data": {
+    "currentUser": {
+      "account": {
+        "aiIssues": {
+          "issues": {
+            "issues": [
+              {
+                "issueId": "5948472b-f7dd-4a6e-aec0-de0390732495",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "team_store_validation_error query result is > 0.0 on 'Team Store Domain Specific Validation Error'"
+                ]
+              },
+              {
+                "issueId": "570c2770-efd2-4c0f-8dc8-fbc1ae69a4df",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "PageViewTiming query deviated from the baseline at least once in 5 minutes on 'Onyx_check_Mark test'"
+                ]
+              },
+              {
+                "issueId": "d056be64-829e-4a83-bfb1-e3232eb3a59b",
+                "priority": "CRITICAL",
+                "state": "CLOSED",
+                "title": [
+                  "Transaction query result is > 2.0 on 'Error fetching commitments (TF)'"
+                ]
+              },
+```
+
+The results can be further filtered by `entityGuid`. 
+
+**Request** 
```

Comment:
don't think you need 'request' in there. it can be implied with the intro to it, like: 

The results can be further filtered by `entityGuid`, as shown in this example call: